tests/integration/security/reachability_test.go

```diff
@@ -44,6 +44,9 @@ func TestReachability(t *testing.T) {
 					ConfigFile: "beta-mtls-on.yaml",
 					Namespace:  systemNM,
 					Include: func(src echo.Instance, opts echo.CallOptions) bool {
+						if src == rctx.HeadlessNaked || opts.Target == rctx.HeadlessNaked {
```

Comment:
Why? This has explicit DR, not auto

Reply:
naked service instance will always fail the mtls test.